app/config/collections.php

```diff
@@ -1440,7 +1440,7 @@
         '$collection' => Database::SYSTEM_COLLECTION_COLLECTIONS,
         '$id' => Database::SYSTEM_COLLECTION_BUCKETS,
         '$permissions' => ['read' => ['*']],
-        'name' => 'File',
+        'name' => 'Bucket',
```

Comment:
I am a bit confused by the `collections.php` and `collections2.php` changes. can you elaborate?

Reply:
Ah with new DB this collections.php should not be required, its for old database, I shall remove this